NEWS.md

```diff
@@ -10,7 +10,17 @@
 
 ### NEW FEATURES
 
-1. New `sort_by()` method for data.tables, [#6662](https://github.com/Rdatatable/data.table/issues/6662). It uses `forder()` to improve upon the data.frame method and also match `DT[order(...)]` behavior with respect to locale. Thanks @rikivillalba for the suggestion and PR.
+1. New `sort_by()` method for data.tables, [#6662](https://github.com/Rdatatable/data.table/issues/6662). It uses `forder()` to improve upon the data.frame method and also match `DT[order(...)]` behavior with respect to locale, and allows sorting by expressions or functions. Thanks @rikivillalba for the suggestion and PR.
+
+```r
+    DT = data.table(a = c(1, 2, 1), b = c(3, 1, 2))
```

Comment:
I would do an example that uses two sorting keys, `~a+b`, to demonstrate how sorting by >1 column works.